docs/cluster_spec.md

```diff
@@ -429,6 +429,25 @@ spec:
     serviceNodePortRange: 30000-33000
 ```
 
+### Customize client-ca file 
+
+This value is passed as `--client-ca-file` for `kube-apiserver`. (default: `/srv/kubernetes/ca.crt`)
+
+```yaml
+spec:
+  kubeAPIServer:
+    clientCAFile: /srv/kubernetes/client-ca.crt
+```
+There are certain cases that the user may want to use a customized client CA file other than the default one generated for Kubernetes. In that case, the user can use this flag to specify the client-ca file to use.
+
+To prepare the customized client-ca file on master nodes, the user can either use the [fileAssets](https://github.com/kubernetes/kops/blob/master/docs/cluster_spec.md#fileassets) feature to push an client-ca file, or embed the customized client-ca file in the master AMI.
```

Comment:
You can just use relevant links in the docs. The preferred location for docs consumption is https://kops.sigs.k8s.io/

Reply:
Thanks, will update.